client/src/components/Markdown/Elements/HistoryDatasetDisplay.vue

```diff
@@ -126,11 +139,17 @@ export default {
     },
     data() {
         return {
+            currentPage: 1,
             expanded: false,
+            perPage: 200,
```

Comment:
I would go with 100 ... https://github.com/bootstrap-vue/bootstrap-vue/issues/4155#issuecomment-534815029
```suggestion
            perPage: 100,
```